dspace-api/src/test/data/dspaceFolder/config/entities/relationship-types.xml

```diff
@@ -0,0 +1,125 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
a dtd or xsd should be provided to facilitate the configuration (and validation)

Reply:
DTD was provided in #2401 (which was merged into this PR).  Closing this request.